### PR TITLE
Rename id to block in goto routing rules

### PR DESCRIPTION
**Pull request by yoganandkunche**
### What is the context of this PR?
Renaming id to block now that we have group

### How to review 
Make sure all the Unit/Integration/functional test are passing

### CODE REVIEW

File: data/en/census_household.json
Comment (shahi645) [inline data/en/census_household.json:3834]: Do we need to update welsh versions?

File: scripts/test_schemas.sh
Comment (shahi645) [inline scripts/test_schemas.sh]: will have to change this back

**Review by shahi645 — CHANGES_REQUESTED** [review]

**Comment by shahi645** [comment]
@yoganandkunche are we doing the yaml files also?

File: data/en/census_household.json
Comment (jonnyshaw89) [reply data/en/census_household.json:3834]: Yes

Comment (yoganandkunche) [reply data/en/census_household.json:3834]: pipenv run ./scripts/test_schemas.sh

File: scripts/test_schemas.sh
Comment (yoganandkunche) [reply scripts/test_schemas.sh]: Will be removed before merge

File: data/en/census_household.json
Comment (shahi645) [reply data/en/census_household.json:3834]: 👍

**Review by shahi645 — APPROVED** [review]
just need to change the test_schemas

File: data/en/census_household.json
Comment (jonnyshaw89) [reply data/en/census_household.json:3834]: i can't see any changes to cy files

Comment (jonnyshaw89) [reply data/en/census_household.json:3834]: Given that test_schemas doesnt actually test the welsh schemas we should add this here.
https://github.com/ONSdigital/eq-survey-runner/blob/92282d2cb76a3ce53b8bb0dbf6b9ca76e5d8047f/scripts/build.sh#L19

Comment (shahi645) [reply data/en/census_household.json:3834]: was meant to say `translate_schemas`

**Review by ajmaddaford — APPROVED** [review]

**Comment by ajmaddaford** [comment]
Blocked until https://github.com/ONSdigital/eq-schema-validator/pull/12 merges

**Pull request merged**
